common/test/unit/com/thoughtworks/go/domain/materials/AbstractMaterialTest.java

```diff
@@ -16,13 +16,13 @@
 
```

Comment:
As discussed we should not have an abstract material test.
Instead, ```hasDestinationFolder ``` must be tested as a behavior of the test cases ```PluggableSCMMaterialTest``` or ```GitMaterialTest``` etc.